### ISSUE TITLE
Issue Notifying Success After Logging in

### ISSUE THREAD

**Issue by cooperhurt** [OfficeDev/microsoft-teams-library-js#669]
I am having a problem when notifying Success after login. Here are the little code snippets:

```
new Promise(async (resolve, reject) => {
			const s = document.createElement("script");
			let r = false;
			s.type = "text/javascript";
			s.src =
				"https://statics.teams.cdn.office.net/sdk/v1.6.0/js/MicrosoftTeams.min.js";
			s.async = true;
			s.onerror = function (err) {
				alert("There was a problem loading Teams Script....");
				console.log(err, "Error");
			};
			s.onload = s.onreadystatechange = async () => {
				if (!r && (!this.readyState || this.readyState == "complete")) {
					r = true;
					resolve();
				}
			};
			const t = document.getElementsByTagName("script")[0];
			t.parentElement.insertBefore(s, t);
		}).then(() => {
		     microsoftTeams.initialize();
		      return new Promise((resolve, reject) => {
			      microsoftTeams.authentication.authenticate({
				      url: window.location.origin + "/neWeb2",
				      width: 600,
				      height: 535,
				      successCallback: () => { alert("Success"); return resolve(this.initialize()); },
				      failureCallback: (err) => { alert("error " + err); reject()}
			      });
		      });
		});
```

Then after logging in I do:
```
	<script type="text/javascript">
		(function () {
			microsoftTeams.authentication.notifySuccess()
		})();
	</script>
```

Then my alert message says:
**"CancelledByUser err"**

I did not close the window, but rather it should be a sucess (it actualy automatically closes the window after the success but reports it back as Cancelled.

Looks like the common problems are:
1. Different domains between the login (not applicalbe, since I'm doing 'window.location.origin'. 
2. Calling Initilize multiple times, possible, I actually removed the initialize before notifySuccess and tried:
```
	<script type="text/javascript">
		(function () {
                      microsoftTeams.initialize();
		      return new Promise((resolve, reject) => {
			     microsoftTeams.authentication.notifySuccess()
		      });
		})();
	</script>
```

Appreciate any insight.

**Comment by Meghana-MSFT** [comment]
We already have a bug raised for microsoftTeams.authentication.notifySuccess() causing CancelledByUser error on desktop app. The concerned team is looking into it.

**Comment by cooperhurt** [comment]
> We have a bug raised for microsoftTeams.authentication.notifySuccess() causing CancelledByUser error on desktop app. Could you please confirm if you are facing this issue on Desktop only?

I actually haven't tried on mobile yet, this is a new app I am creating, as of right now I am jsut trying this in a website browser. And then manually testing this in a website browser.

**Comment by Meghana-MSFT** [comment]
Can you please confirm if you are facing this issue on desktop as well?

**Comment by cooperhurt** [comment]
> Can you please confirm if you are facing this issue on desktop as well?

Correct, do you know how long out a fix to something like this might be? Or a version of the SDK that it doesn't exist in?

**Comment by Meghana-MSFT** [comment]
We do not have any ETA for the fix? Also could you please share console logs from web where you get this issue?

**Comment by vvolodin** [comment]
I'm having the same issue on Desktop only. I've also tested Android and web - works fine there.

**Comment by Meghana-MSFT** [comment]
@cooperhurt - Please share logs if you are still facing the issue on web? Also Please be informed that a bug has already been raised for desktop.

**Comment by cooperhurt** [comment]
@Meghana-MSFT Happy to provide more logs. Not sure how, I set a debugger, the only thing it shows is 'Cancelled By User' on the error. Even after a successfull login.

![image](https://user-images.githubusercontent.com/40371983/135928301-b01c20bb-1fdf-48b5-bd51-a78deb2c87ae.png)

**Comment by Meghana-MSFT** [comment]
@cooperhurt - While checking it in browser check Dev tools and share logs from there.

**Comment by cooperhurt** [comment]
Intresting, so I was able to test this inside my AppStudio.

When testing in App Studio I gett 'FailedToOpenWindow', but when testing this page outside of Teams I get 'CancelledByUser'.

Not sure why it is showing 'FailedToOpenWindow', errors/debugging seem to lack a little with the SDK.

**Comment by Meghana-MSFT** [comment]
Could you please try downgrading the version to 1.3.x and check. An ICM has already been raised for the same and the team is working on the fix. We will update you on the same.
